rocAL/rocAL/source/augmentations/geometry_augmentations/node_crop_mirror_normalize.cpp

```diff
@@ -26,59 +26,91 @@ THE SOFTWARE.
 #include "exception.h"
 
 
-CropMirrorNormalizeNode::CropMirrorNormalizeNode(const std::vector<Image *> &inputs,
-                                                 const std::vector<Image *> &outputs) :
+CropMirrorNormalizeNode::CropMirrorNormalizeNode(const std::vector<rocalTensor *> &inputs,
+                                                 const std::vector<rocalTensor *> &outputs) :
         Node(inputs, outputs),
-        _mirror(MIRROR_RANGE[0], MIRROR_RANGE[1])
-{
+        _mirror(MIRROR_RANGE[0], MIRROR_RANGE[1]) {
         _crop_param = std::make_shared<RocalCropParam>(_batch_size);
 }
 
-void CropMirrorNormalizeNode::create_node()
-{
+void CropMirrorNormalizeNode::create_node() {
     if(_node)
         return;
 
     if(_crop_param->crop_h == 0 || _crop_param->crop_w == 0)
         THROW("Uninitialized destination dimension - Invalid Crop Sizes")
 
     _crop_param->create_array(_graph);
-    _mean_vx.resize(_batch_size);
-    _std_dev_vx.resize(_batch_size);
-    for (uint i=0; i < _batch_size; i++ ) {
-         _mean_vx[i] = _mean;
-         _std_dev_vx[i] = _std_dev;
+
+    std::vector<float> mean_vec, std_dev_vec;
```

Comment:
looks like mean and std are converted to multiplier and offset. Please rename the variables to reflect that. Also we need to pass it using a single tensor if multiplier and offset are different for each image in tensor.